### PR TITLE
style: tweak styling for about HTML content

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -60,15 +60,35 @@ body {
   color: #91caff;
 }
 
+
+/* BEGIN about HTML styling */
+
+/*
+Header elements which are the first child of the about content container shouldn't have the default Ant Design top
```
Comment (SanjeevLakhwani) [inline src/styles.css:67]: The about html does not inherit antd styles for what I know so this shouldnt be an issue.

File: src/styles.css [hunk 2]
```diff
@@ -60,15 +60,35 @@ body {
...
+margin
+ */
+.about-content h2:first-child, h3:first-child {
```
Comment (SanjeevLakhwani) [inline src/styles.css:70]: why just h2 and h3, why not h1?

File: src/styles.css (hunk 1 above)
Comment (davidlougheed) [reply src/styles.css:67]: it does seem to on my end and is visible for example with RENATA: https://renata.bento.sd4h.ca/en/

File: src/styles.css (hunk 2 above)
Comment (davidlougheed) [reply src/styles.css:70]: was just doing what is used but you're right, should be all

File: src/styles.css [hunk 3]
```diff
@@ -60,15 +60,32 @@ body {
   color: #91caff;
 }
 
+
+/* BEGIN about HTML styling */
+
+/* Header elements which are the first child of the about content container shouldn't have the default top margin */
+.about-content h1:first-child, h2:first-child, h3:first-child, h4:first-child, h5:first-child, h6:first-child {
```
Comment (SanjeevLakhwani) [inline src/styles.css:67]: We can also consider maybe just affecting the first child of about content instead of just the headers as first child?

**Review by SanjeevLakhwani — COMMENTED** [review]
lgtm, just the comment

File: src/styles.css (hunk 3 above)
Comment (davidlougheed) [reply src/styles.css]: done

**Pull request merged**